src/Groups/gsets.jl

```diff
@@ -41,7 +41,7 @@ The fields are
     action_function::Function
     seeds
 
-    function GSetByElements(G::T, fun::Function, seeds; closed::Bool = false, check::Bool = true) where {T<:Union{GAPGroup, FinGenAbGroup}}
+    function GSetByElements(G::T, fun::Function, seeds; closed::Bool = false, check::Bool = true) where {T<:Union{AbstractAlgebra.Group, FinGenAbGroup}}
```

Comment:
```suggestion
    function GSetByElements(G::T, fun::Function, seeds; closed::Bool = false, check::Bool = true) where {T<:Union{Group, FinGenAbGroup}}
```
You don't need this qualification anymore (here and at other places)
You may need to merge master into this branch or rebase for it to work